rcsb/utils/dictionary/DictMethodEntryHelper.py

```diff
@@ -570,6 +571,15 @@ def buildContainerEntryIds(self, dataContainer, catName, **kwargs):
             tV = ",".join([str(tId) for tId in modelIdList]) if modelIdList else "?"
             cObj.setValue(tV, "model_ids", 0)
             #
+            if dataContainer.exists("citation"):
+                tObj = dataContainer.getObj("citation")
+                for ii in range(tObj.getRowCount()):
+                    pv = tObj.getValue("id", ii)
+                    pm = tObj.getValue("pdbx_database_id_PubMed", ii)
+                    if pv.upper() == "PRIMARY" and pm:
+                        cObj.setValue(pm, "pubmed_id", 0)
```

Comment:
Have you tested the following cases?:
- `pdbx_database_id_PubMed` attribute is completely missing from the data container
- `pdbx_database_id_PubMed` is present but is set to `.`